stdlib/inspect.pyi

```diff
@@ -144,6 +144,10 @@ class Signature:
     else:
         @classmethod
         def from_callable(cls, obj: Callable[..., Any], *, follow_wrapped: bool = ...) -> Signature: ...
+    @classmethod
+    def from_builtin(cls, func: Callable[..., Any]) -> Signature: ...
```

Comment:
Do you have a specific reason for adding these? They're deprecated, so we might as well give people a service by not putting them in typeshed.

Reply:
No reason beyond completionism! I'll get rid of em